### ISSUE TITLE
List of supported clients?

### ISSUE THREAD

**Issue by psugihara** [SparkPost/heml#1]
Cool project. We're going through some email hell right now for a new set of templates and would love to use something like this next time.

What clients are being targeted... aol, yahoo, outlook 2010/2013/2016? How is this tested?

**Comment by avigoldman** [comment]
Hey @psugihara! 
You ask a really important question. Thus far its been a lot of manual testing for the client support. I'm working on some ideas for automating/building out tests. I want to do snapshot tests for each of the element and possibly automate a test with email on acid for getting visual checks. Any and all ideas are welcome.

**Comment by dylmye** [comment]
[Litmus](https://litmus.com/) seems to be the big player in this field, like the Browserstack of email client testing, but they don't offer FOSS discounts and it's quite expensive. I believe you can test the email client in Browserstack however?

**Comment by avigoldman** [comment]
Yeah, Litmus is great but pricey. Email on Acid offers (I think) everything we need to test this well, and is dramatically cheaper. I've never seen Testi@, so I'll definitely dig into that!

**Comment by hellocosmin** [comment]
@avigoldman I can help with testing in Email on Acid. DM me in #emailgeeks on Slack if you want, and we take it from there ;)

**Comment by avigoldman** [comment]
Sweet! Lets do that!
For context:

**Comment by avigoldman** [comment]
Sweet, let's develop a basic plan in there that we can bring back to this issue to get more eyes on it!

For context: The [#emailgeeks](https://email.geeks.chat/) slack is an amazing community of people involved in basically every area of email, from development to delivery. Definitely, check it out 💌
